### DISCUSSION TITLE
is there a way to automatically update `lastmod` in .md file?

### DISCUSSION THREAD

**Discussion by xuchen** [thuliteio/doks#1233] [Q&A] [answered]
Every time we save and publish a file, auto update it to the newest datetime, thank you!

**Comment by h-enk** [comment, marked as answer]
Use `git` dates — see: [Last Modified Date with Hugo](https://www.andrewjstevens.com/posts/2021/03/last-modified-date-with-hugo/)

**Reply by xuchen** [reply]
it works, thanks!
